Backpack/Gradient/Classes/BPKGradientLayer.m

```diff
@@ -0,0 +1,88 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKGradientLayer.h"
+
+#import "BPKGradient.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@implementation BPKGradientLayer
+
+- (instancetype)init {
+    self = [super init];
+
+    if (self) {
+        self.needsDisplayOnBoundsChange = YES;
+    }
+
+    return self;
+}
+
++ (BOOL)needsDisplayForKey:(NSString *)key {
+    if ([key isEqualToString:@"gradient"]) {
```

Comment:
instead of the string "gradient" you could say `NSStringFromSelector(@selector(gradient))`

Reply:
Noice :)